### ISSUE TITLE
[Suggestions] New Crystal Mechanics improvements

### ISSUE THREAD

**Issue by DemonInTheCloset** [HellFirePvP/AstralSorcery#1626]
**Maxed out Attributes/Crystals should have an indication**
Coming from 1.12 Astral, Size maxing out at III is confusing, this is exacerbated by Purity and other attributes maxing out at II, so I suggest having a color change in the number when it reaches it's maximum value.
Same thing with Crystals, having a way of knowing when it can't fit any more attributes would make it more intuitive. Suggestion, have a tooltip (ie. _This crystal seems to be too unstable to improve_) or a color change on the crystal name.

**Size and Tool Durability should be mutually exclusive:**
I expect the size to affect the durability of a too made out of crystals, that's why I think that Tool Durability is redundant.

I created this comment on DW20's latest video of his 1.16.4 Let's Play, and though it would be helpful to elaborate on it here.

> My opinion about the new Astral Crystal Mechanics:
> 
> tl;dr: Overall big improvement. Sad about the limited automation options
> 
> Good things about previous Astral Crystals:
> 1. Besides cutting, everything else was fully automatable
> 2. Stats were more straightforward & had clear maximums
> 3. You could repeatedly get perfect crystals
> 
> Bad things about previous Crystals:
> 1. The process of getting a perfect crystal was grindy and not fun unless you, like dire, had fun automating them
> 2. Even when fully automated, you still had to cut them by hand and grow them back up to full size.
> 
> Good things about new implementation:
> 1. You can't make a "perfect" crystal, you have to tailor make them for each purpose.
> 2. There is a clear way to make crystals with the stats that you want, but it isn't trivial to do.
> 3. The stats are small numbers.
> 
> Things that can be improved about the new implementation:
> 1. A maxed out attribute should have an indication of that (like the color change in old astral, ie. change the numerals from orange to green/red)
> 2. A crystal should indicate that its "full" (has all its slots filled), suggestion, change crystal name color.
> (Dire didn't read the astral book on camera so maybe its there, but)
> 3. It's difficult to know what each of the new attributes does.
> 4. I expect the Size attribute to impact tool durability, but there is a Tool durability attribute, and that makes size kinda redundant imo.
